docs/modules/develop/pages/classes/cells.adoc

```diff
@@ -0,0 +1,44 @@
+= Cells
+
+As explained in dedicated page xref:develop:view_models_aka_cells.adoc[View Models (Cells)], a cell is an object that represent a fragment of your UI.
+
+In order to define a cell, you need to create a class that inherits from `Decidim::ViewModel` in your plugin.
+
+Cells classes are located in the `app/cells/decidim/<my_plugin>` directory, and named: `<my_cell>_cell.rb`.
+
+For example, if you want to create a cell that renders a simple text:
+
+```ruby
+# frozen_string_literal: true
+
+# app/cells/decidim/my_module/my_cell_cell.rb
+module Decidim
+  module MyPlugin
```

Comment:
```suggestion
  module MyModule
```

This needs to be changed everywhere. If I do a ctrl+f in this page in GH (the "Files changed" tab of the PR) I should not find plugin never

Reply:
Will do in a standalone commit.

Reply:
Fixed in c010e4fed7